test/model.create.test.js

```diff
@@ -49,31 +49,31 @@ describe('model', function() {
     });
 
     it('fires callback when passed 0 docs', function(done) {
-      B.create(function(err, a) {
+      testModel.create(function(err, a) {
         assert.ifError(err);
         assert.ok(!a);
         done();
       });
     });
 
     it('fires callback when empty array passed', function(done) {
-      B.create([], function(err, a) {
+      testModel.create([], function(err, a) {
         assert.ifError(err);
         assert.deepEqual(a, []);
         done();
       });
     });
 
     it('supports passing options', async function() {
-      const docs = await B.create([{}], { validateBeforeSave: false });
+      const docs = await testModel.create([{}], { validateBeforeSave: false });
 
       assert.ok(Array.isArray(docs));
       assert.equal(docs.length, 1);
     });
 
     it('returns a promise', function() {
-      const p = B.create({ title: 'returns promise' });
-      assert.ok(p instanceof mongoose.Promise);
+      const returnPromise = testModel.create({ title: 'returns promise' });
+      assert.ok(returnPromise instanceof mongoose.Promise);
```

Comment:
I would expect a variable with a verb to be a function, perhaps `promise`/`value`?